Frontend/src/components/Register.js

```diff
@@ -0,0 +1,245 @@
+    criptoWallet: Yup.string()
+        .required('The CriptoWallet address is required')
+        .min(8, 'The CriptoWallet address must be exactly 8 characters')
+        .max(8, 'The CriptoWallet address must be exactly 8 characters'),
+    acceptTerms: Yup.bool().oneOf([true], 'Accept Terms is required')
+  });
+
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors }
+  } = useForm({
+    resolver: yupResolver(validationSchema)
+  });
+
+  const onSubmit = userData => {
+
+      AuthService.register(userData).then(
+          (response) => {
+            setMessage(response.data.message);
+            setSuccessful(true);
+            history.push("/activeCripto")
+            window.location.reload();
+          },
+          (error) => {
+            const resMessage =
+                (error.response &&
+                    error.response.data &&
+                    error.response.data.message) ||
+                error.message ||
+                error.toString();
+
+            setMessage(resMessage);
+            setSuccessful(false);
+          }
+      );
+    };
+
+  return (
+      <div className="register-form">
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className="form-group">
+            <label>Name</label>
+            <input
+                name="name"
+                type="text"
+                {...register('name')}
+                className={`form-control ${errors.name ? 'is-invalid' : ''}`}
+            />
+            <div className="invalid-feedback">{errors.name?.message}</div>
+          </div>
+
+          <div className="form-group">
+            <label>Surname</label>
+            <input
+                name="surname"
+                type="text"
+                {...register('surname')}
+                className={`form-control ${errors.surname ? 'is-invalid' : ''}`}
+            />
+            <div className="invalid-feedback">{errors.surname?.message}</div>
+          </div>
+
+          <div className="form-group">
+            <label>Username</label>
+            <input
+                name="userName"
+                type="text"
+                {...register('userName')}
+                className={`form-control ${errors.userName ? 'is-invalid' : ''}`}
+            />
+            <div className="invalid-feedback">{errors.userName?.message}</div>
+          </div>
+
+          <div className="form-group">
+            <label>Email</label>
+            <input
+                name="email"
+                type="text"
+                {...register('email')}
+                className={`form-control ${errors.email ? 'is-invalid' : ''}`}
+            />
+            <div className="invalid-feedback">{errors.email?.message}</div>
+          </div>
+
+          <div className="form-group">
+            <label>Address</label>
+            <input
+                name="address"
+                type="text"
+                {...register('address')}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JeremiasFuentes/unq-desapp-fuentes-diaz/pullRequest?prid=8198099)
```suggestion
                {...register("address")}
```